app/models/state_file_nj_intake.rb

```diff
@@ -114,7 +114,7 @@
 class StateFileNjIntake < StateFileBaseIntake
   self.ignored_columns += ["primary_signature_pin", "spouse_signature_pin"]
   encrypts :account_number, :routing_number, :raw_direct_file_data, :raw_direct_file_intake_data
-  has_one :state_file_nj_analytics
+  has_many :state_file_nj_analytics
```

Comment:
[dust] This might be my ignorance of ActiveRecord, but are we "allowed" to change the model file directly for this kind of relation, or this that something we should have a migration for?

Reply:
yeah, should be fine to change. the db structure is the same, we're just letting rails know to look for many potential associated records instead of strictly retrieving zero or one.